### PR TITLE
Change versions to deployments and add order query param to deployments call

### PR DESCRIPTION
**Pull request by jspspike**

### CODE REVIEW

File: packages/wrangler/src/deployments.ts [hunk 1]
```diff
@@ -40,8 +41,11 @@ export async function deployments(
 	);
 
 	const scriptTag = scriptMetadata.default_environment.script.tag;
+	const params = new URLSearchParams({ order: "asc" });
 	const { items: deploys } = await fetchResult<DeploymentListRes>(
-		`/accounts/${accountId}/workers/versions/by-script/${scriptTag}`
+		`/accounts/${accountId}/workers/deployments/by-script/${scriptTag}`,
+		undefined,
```
Comment (danielrs) [inline packages/wrangler/src/deployments.ts:47]: What is this `undefined` (just curious)?

**Review by danielrs — APPROVED** [review]

File: packages/wrangler/src/deployments.ts (hunk 1 above)
Comment (jspspike) [reply packages/wrangler/src/deployments.ts:47]:
``` javascript
export async function fetchResult<ResponseType>(
	resource: string,
	init: RequestInit = {},
	queryParams?: URLSearchParams,
	abortSignal?: AbortSignal
): Promise<ResponseType>
```
& is the function signature for `fetchResult`. You can pass `init` for your request content but I'm not doing that so leaving it as `undefined`.
There might be a way to not specify it all using named parameters but I couldn't see how to do that

**Review by rozenmd — APPROVED** [review]
LGTM 👍

File: packages/wrangler/src/deployments.ts (hunk 1 above)
Comment (JacobMGEvans) [reply packages/wrangler/src/deployments.ts:47]: Yeah this is JavaScript idiosyncratic behavior for Web APIs lol

File: packages/wrangler/src/deployments.ts [hunk 2]
```diff
@@ -52,8 +56,8 @@ Author: ${versions.metadata.author_email}
 Source: ${sourceStr(versions.metadata.source)}\n`
 	);
 
-	versionMessages[0] += "🟩 Active";
-	logger.log(...versionMessages.reverse());
+	versionMessages[versionMessages.length - 1] += "🟩 Active";
```
Comment (JacobMGEvans) [inline packages/wrangler/src/deployments.ts:59]: This can also be `versionMessages.at(-1)`

**Pull request merged**